### ISSUE TITLE
Add warning for set-touch xxx fixed

### ISSUE THREAD

**Issue by Amolith** [drduh/YubiKey-Guide#197]
In the [Require touch](https://github.com/drduh/YubiKey-Guide#require-touch) section, it would be good to either add a warning about not being able to disable the touch requirement if `fixed` is used or change the default to `on`. It would also be beneficial to tell users to run `ykman openpgp set-touch -h` to see what other options there are depending on their use-case.

A lot of people will probably run that command without reading the help page expecting that they would be able to change it later if needed (like me). The current state would put them in a frustrating situation where they have to reset the slot and go through the process of importing the keys again or continue using it as-is even though it might be a huge hassle.

**Comment by Amolith** [comment]
I feel like some explanation is necessary for why this is such a pain point for me. I recently set up [OfflineIMAP](https://www.offlineimap.org/) and my passwords are encrypted with my GPG key. When OfflineIMAP syncs with my mail providers, it does so incrementally and I have to touch my YubiKey every second or two while it goes through my 20-odd accounts. I then have to repeat this the next time it syncs. My systemd timer is currently set to run the program every hour but I would really rather have it run every 15-20 minutes to get emails in a more timely manner. However, requiring interaction every few seconds every 15 minutes is ridiculous. 

I'm going to reset the slot and configure it with `cached` but I opened the issue so that future users are less likely to run into the same problem.
